86BoxManager/Program.cs

```diff
@@ -54,10 +58,15 @@ public struct COPYDATASTRUCT
         [STAThread]
         static void Main()
         {
+            // Windows Vista+
             if (Environment.OSVersion.Version.Major >= 6)
```

Comment:
Since 86Box itself doesn't support anything earlier than Windows 7 SP1, these checks will be redundant once the Manager project target's .NET Framework 4.8 (which no longer supports Vista) instead of 4.6.

Reply:
for this, surely it is prudent to just close at start if we are running on Windows 7 SP1 until we target 4.8?